pkg/model/components/cilium.go

```diff
@@ -139,6 +139,10 @@ func (b *CiliumOptionsBuilder) BuildOptions(o interface{}) error {
 		c.EnableL7Proxy = fi.Bool(true)
 	}
 
+	if c.DisableCNPStatusUpdates == nil {
+		c.DisableCNPStatusUpdates = fi.Bool(false)
+	}
+
```

Comment:
This should be the only change to this file, right?

Reply:
VS Code started autoformatting some new stuff on save. I just let it. Seems to be improvements (I am really inconsistent when it comes to whitespace usage, so I like this).

Reply:
Was just thinking that this would make cherry-picks harder later.

Reply:
Probably not. But anyways, removed those changes now